080-Recipes/010-csv-data/010-import-csv-data-UI.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Import CSV data
+navTitle: Import CSV data
+keywords: ['csv', 'import', 'migrate data']
+description: Import records from a local CSV file into a Xata database table
+slug: recipes/import-data
+published: true
+---
+
+## Overview
+
+A CSV file is a standard text file where data fields are typically divided by commas (,), and saved with the `.csv` extension.
+You can import and work with CSV file data in your Xata database.
+
+**NOTE:** While CSV files are explicity mentioned, Xata supports the import of various file formats, including tab-separated values (TSV), comma-separated values (CSV), delimiter-separated values (DSV), and plain text data. This covers a range of extensions including `.tsv`, `.csv`, `.dsv`, and `.txt` formats.
```

Comment:
This might be worth mentioning somewhere, but I'm wondering if it's a bit too prominent at the moment?